<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>823</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>563</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>633</width>
    <height>823</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8.5pt &quot;Lucida Console&quot;;
gridline-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Código de vuelo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pronosticador(a)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pronosticador">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de documento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numero">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="vuelos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hora de emisión (UTC)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emision">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hora del pronóstico de despegue (UTC)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="despegue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>436</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Direccion del viento en superficie (grados)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="direccion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad del Viento (nudos)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="velocidad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperatura (ºC)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="temperatura">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de vuelo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="alto">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alto</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bajo">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bajo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mapa de tiempo significante</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="actual_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual - 6hrs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="actual">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rango de validez de los mapas de V y T</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="z12">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+12 hr</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="z24">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+24 hr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crea">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>640</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Carpeta</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="z6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+06 hr</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="z18">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+18 hr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SOLO PARA VUELOS BAJOS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="zygrid">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Usar mapa Zygrib</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin50">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin100">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin180">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin240">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>540</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Factor gris mapa 50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Factor gris mapa 100 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>500</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Factor gris mapa 180</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Factor gris mapa 240</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboQNH">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>QNH (inHg)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="descargar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>600</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Descargar Mapas</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="actual_12">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual - 12hrs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="actual_18">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual - 18hrs</string>
    </property>
   </widget>
   <widget class="QComboBox" name="aeropuerto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aeropuerto</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sigmet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Texto adicional (SIGMET)</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
